database/src/migrations/release.0.34/api_get_eml_data_package.sql

```diff
@@ -294,7 +294,7 @@ begin
         <taxonomicClassification>
 				  <taxonRankName>%1$s</taxonRankName>
 				  <taxonRankValue>%2$s</taxonRankValue>
-				  <commonName>%3$s</commonName>
+				  <commonNames>%3$s</commonNames>

```

Comment:
Looks like this just got caught in the find/replace.

Since this is an old migration, we should undo this change.

Reply:
Oops, I'll make sure to exclude migrations when find-replacing. Thanks